identity/did/bootstrapper.go

```diff
@@ -48,7 +48,14 @@ func (*Bootstrapper) Bootstrap(context map[string]interface{}) error {
 	service := NewService(cfg, factoryContract, client)
 	context[BootstrappedDIDService] = service
 
-	identity := NewIdentity(cfg, client)
+	//TODO config needs to store correct did format
+	id, err := cfg.GetIdentityID()
+	if err != nil {
+		return err
+	}
+	did := NewDID(common.BytesToAddress(id))
+
+	identity := NewIdentity(cfg, client, &did)
```

Comment:
There will be multiple identities under the same host. Why do we need to bootstrap here? Is this `identity` only used by the p2p layer? In which case we should rename it.

Reply:
I thought multi-account is abstracted in that way, that the config already includes the identity address of the current account. 

I thought the bootstrapper would be called for every different identity in the multi-account scenario. If not I will remove it.

I added it to the bootstrapper to easier use the identity later on.

Reply:
> I thought multi-account is abstracted in that way, that the config already includes the identity address of the current account.

Not at bootstrap level. Accounts are dynamically created, loaded per request basis.

> I thought the bootstrapper would be called for every different identity in the multi-account scenario. If not I will remove it.

Nope, bootstrapper is only called at start up time. Then dynamically load the accounts per request. I would remove this as it is confusing. Within your code you should use the `context.Context` to identify the ID to use.

Reply:
I already looked it up. Thanks for the clarification.

Reply:
I will add the changes for context in the next PR